hypervisor/boot/sbl/multiboot.c

```diff
@@ -116,6 +123,26 @@ static void *get_kernel_load_addr(void *kernel_src_addr)
 	return kernel_src_addr;
 }
 
+static void get_emmc_serial(char *cmdline, uint8_t *serial)
+{
+	char *arg;
+	char *param;
+	uint32_t len;
+
+	len = strnlen_s("androidboot.serialno=", MEM_2K);
```

Comment:
why ABL names it with "androidboot"

Reply:
This should be for Android kernel, for SOS, they should change the name. I guess they keep it the same for Native and ACRN. Do we need to request them to change it?

Reply:
you can check with Kai.

Reply:
I'm fine. if ACRN team has concern, we can bring it to ABL team